fe/fe-core/src/main/java/com/starrocks/common/proc/StreamLoadsLabelProcDir.java

```diff
@@ -0,0 +1,42 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
```

Comment:
I have a question. If we have the begin/prepare/commit/rollback already, when to use the feature?